templates/csharp/custom-copilot-basic/README.md.tpl

```diff
@@ -0,0 +1,84 @@
+# Overview of the AI Chat Bot template 12345
```

Comment:
Remove the tailing `12345`? By the way, seems this readme file is same with the other readme under `.{{NewProjectTypeName}}/README.md.tpl`. Are both of them required?

Reply:
`{{NewProjectTypeName}}/README.md.tpl` is required only. Fixed.